cms/static/js/factories/manage_users.js

```diff
@@ -9,7 +9,7 @@ function(_, gettext, ManageUsersAndRoles) {
             var local_messages = _.extend({}, messages);
             local_messages.alreadyMember.title = gettext('Already a course team member');
             local_messages.deleteUser.messageTpl = gettext(
-                'Are you sure you want to delete {email} from the course team for “{container}”?'
+                'Are you sure you want to delete <%= email %> from the course team for “<%= container %>”?'
```

Comment:
If we do go with the traditional underscore interpolation syntax, we should use `<%-` where appropriate.